LayoutTests/accessibility/mac/textmarker-routines.html

```diff
@@ -35,7 +35,8 @@
 
         shouldBeTrue("item2.isTextMarkerValid(secondTextMarker)");
         shouldBe("item2.indexForTextMarker(secondTextMarker)", "5");
-        shouldBeTrue("item2.textMarkerForIndex(item2.indexForTextMarker(secondTextMarker)).isEqual(secondTextMarker)");       
+        shouldBeTrue("item2.textMarkerForIndex(item2.indexForTextMarker(secondTextMarker)).isEqual(secondTextMarker)");
+
```

Comment:
Seems like we added an unnecessary new line here?

Reply:
Yes, I removed the trailing spaces in this line of code and ended up adding an extra blank line while debugging test failures :-). Deleted a few unnecessary blank lines while at it. This test needs to be brought to AX standard but that's a matter for another patch.